roles/default/kiali-deploy/defaults/main.yml

```
@@ -187,6 +187,10 @@ kiali_defaults:
 
   kiali_feature_flags:
     istio_injection_action: true
+    ui_defaults:
+      namespaces: []
+      metrics_per_refresh: "1m"
+      auto_refresh_interval: "15s"
```

Only request I have is to make sure the peer settings are alphabetically ordered:
```suggestion
    ui_defaults:
      auto_refresh_interval: "15s"
      metrics_per_refresh: "1m"
      namespaces: []
```